### DISCUSSION TITLE
xDrip+ - web follower not working on pixel 7a

### DISCUSSION THREAD

**Discussion by KleinRiese** [NightscoutFoundation/xDrip#3411] [Q&A]
Hi there,

I have been using xDrip on my old Galaxy A41 for a year without any problems. Yesterday I upgraded my phone and can't get xDrip to work on it. My new phone is a Pixel 7a (fresh install on Android 13 - not working, no changes after upgrade to Android 14).

- xDrip 2024.03.27 as web follower (libre)
- Tried a fresh install
- Tried to import all settings with qr code
- Tried a full backup

xDrip doesn't get any readings, I get the message "BG Reading Unavailable, xDrip Data Collection Service is running" in the notification bar.

Do you have any tips?

**Comment by PhilTU** [comment]
Look at the System Status page for Web Follow (swipe sideways from Classic) to see if there are clues in the error message. Similarly error/event log may have something. You are following a Libre 2/3 via Librelinkup method Ithink - did you try LibreLinkUp app on same phone to confirm username/password and connectivity ?

**Reply by KleinRiese** [reply]
Sorry, I forgot to mention - you are correct, I'm following a Libre 3. LibreLinkUp is working flawless.

I only have those 2 Lines in the log:
![Screenshot_20240329-121544](https://github.com/NightscoutFoundation/xDrip/assets/83984038/96856a6d-9e0c-4d56-9677-bbe90929cc4e)

And this is shown on the status page:
(Latest reading 15 hours is because I used a backup from my old phone this time)
![Screenshot_20240329-121641](https://github.com/NightscoutFoundation/xDrip/assets/83984038/a2f954d2-1ef3-45af-86d2-c28e4726a09c)

**Reply by PhilTU** [reply]
ok, classic status page ?

you can also add tags to the event log to get more verbosity - WebFollower:v 

this goes in Settings / Extra tags for logging https://xdrip.readthedocs.io/en/latest/use/lesscommon/#extra-logging-settings

you might also like to try force stopping LibreLinkUp if it's on the same phone in case there is a rate of requests issue or a clash.

**Reply by KleinRiese** [reply]
Classic page:
![Screenshot_20240329-125122](https://github.com/NightscoutFoundation/xDrip/assets/83984038/ff029563-a47d-4cc4-a6ca-aa645771157d)

And verbose logging - seems data is missing, but the request works? At least it gets my region.
![Screenshot_20240329-125316](https://github.com/NightscoutFoundation/xDrip/assets/83984038/33cfe9c6-e4f1-4a14-bdc8-4fdb23d9e5b6)

**Reply by PhilTU** [reply]
was there nothing below that on Classic status ? Web follower status also usually shows any errors (I'm always nervous when screens are cropped)

yes we can see it requested and got a result. There is a trick to refresh the script where if you add a character to beonlabs in the Web Follower settings then go back to the main screen then change it back in settings it forces a refresh. 

Edit : I looked it up and jamorham said "can you also try toggling the proxy support on/off and see if this does anything as this resets some of the internals. "

if you also add another parameter you may see more WebFollower:v,WebFollow:v there is a comma "," between the two tags

**Reply by KleinRiese** [reply]
No, only 2 buttons "restart collector" were below the status. No Error noted there.
I also tried changing the script / updating beonlabs, no change at all.

After that, I gave up for today and kept minding my normal business. And guess what?

I went to the cinema this afternoon and after a good 90 minutes my phone suddenly rang with a signal loss message. Since then, everything has been working as it should. So somehow, 6 hours after my last change, things have been sorted. I also rebooted the phone - still working. This is a bit unsatisfactory for me now, because it looks like a user error ;)

Anyway - many thanks for your help!

**Reply by PhilTU** [reply]
No problem. It's also possible the server end was the problem.
